source/MaterialXTest/MaterialXGenGlsl/GenGlsl.cpp

```diff
@@ -139,25 +139,26 @@ static void generateGlslCode()
     testRootPaths.push_back(testRootPath3);
     const mx::FilePath libSearchPath = mx::FilePath::getCurrentPath() / mx::FilePath("libraries");
     const mx::FileSearchPath srcSearchPath(libSearchPath.asString());
-    const mx::FilePath logPath("genglsl_glsl400_generate_test.txt");
-
     bool writeShadersToDisk = false;
-    GlslShaderGeneratorTester tester(mx::GlslShaderGenerator::create(), testRootPaths, libSearchPath, srcSearchPath, logPath, writeShadersToDisk);
 
     const mx::GenOptions genOptions;
     mx::FilePath optionsFilePath = testRootPath / mx::FilePath("_options.mtlx");
-    tester.validate(genOptions, optionsFilePath);
-
-    // Test glsl 400 with uniform layout qualifier extensions
-    const mx::FilePath glsl42LogPath("genglsl_glsl420_layout_generate_test.txt");
-    writeShadersToDisk = false;
-    GlslShaderGeneratorTester testerglsl42(mx::GlslShaderGenerator::create(), testRootPaths, libSearchPath, srcSearchPath, glsl42LogPath, writeShadersToDisk);
-    
-    // enable tester to use userdata
-    testerglsl42.validate(genOptions, optionsFilePath, true);
+
+    const bool generateLayout = standardGLSL ? false : true;
```

Comment:
Would be prettier code with: `const bool generateLayout = !standardGLSL;

Or even better just name the first variable `generateLayout`? So the function becomes:

static void generateGlslCode(bool generateLayout = false)

Reply:
I had kept it separate because the semantics were different, but I agree it's a little verbose.